packages/text-bitmap/src/BitmapFontLoader.ts

```diff
@@ -16,16 +16,6 @@ import type { Texture } from '@pixi/core';
  */
 export class BitmapFontLoader
 {
-    /**
-     * Called when the plugin is installed.
-     *
-     * @see PIXI.Loader.registerPlugin
-     */
-    public static add(): void
-    {
-        LoaderResource.setExtensionXhrType('fnt', LoaderResource.XHR_RESPONSE_TYPE.DOCUMENT);
```

Comment:
I think we should keep this `add()` method but parse `fnt` as `LoaderResource.XHR_RESPONSE_TYPE.TEXT` instead. Would be nice to be explicit and not assume the Loader will do the right thing here and default to text.

Reply:
Done